clickplc/tests/test_driver.py

```diff
@@ -46,11 +90,13 @@ async def test_ds_roundtrip(plc_driver):
 
 @pytest.mark.asyncio
 async def test_get_error_handling(plc_driver):
-    with pytest.raises(ValueError, match='End address must be greater than start address.'):
+    with pytest.raises(ValueError, match='An address must be supplied'):
+        await plc_driver.get()
+    with pytest.raises(ValueError, match='End address must be greater than start address'):
         await plc_driver.get('c3-c1')
     with pytest.raises(ValueError, match='foo currently unsupported'):
         await plc_driver.get('foo1')
-    with pytest.raises(ValueError, match='Inter-category ranges are unsupported.'):
+    with pytest.raises(ValueError, match='Inter-category ranges are unsupported'):
```

Comment:
War on periods. Preference here?

Reply:
match is regex so period would match more than just a period

Reply:
Ohh, yeah completely missed that.